From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Thu, 4 May 2023 08:36:51 +0200
Subject: [PATCH] [http.c] Remove #include <ctype.h>

This simply removes the include statement for ctype.h. opusfile is not
very happy about our ctype.h because it contains static inline
definitions.

Also removing it does not hurt since we can compile fine without it.
---
 src/http.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/http.c b/src/http.c
index bd0856270a2108db175f4c00508e221e616801dd..aca5b381df0fd2b4f18ad45a9f446a098d57c9f6 100644
--- a/src/http.c
+++ b/src/http.c
@@ -14,7 +14,6 @@
 #endif

 #include "internal.h"
-#include <ctype.h>
 #include <errno.h>
 #include <limits.h>
 #include <string.h>
